<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Films</class>
 <widget class="QDialog" name="Films">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nouveau Film</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit, QComboBox {
    background-color: #3c3c3c;
    color: white;
    border: 1px solid #555;
}
QPushButton {
    background-color: #555;
    color: white;
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #777;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="lblTitre">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Titre :</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="btnAnnuler">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblDuree">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Durée :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" alignment="Qt::AlignVCenter">
    <widget class="QLabel" name="lblCategorie">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Catégorie :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="txtTitre"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="txtDuree"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="txtCategorie"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="btnEnregistrer">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Enregistrer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
